webapp/src/shared/components/Button/Button.css

```
@@ -65,3 +65,10 @@
   position: absolute;
   left: 0.675rem;
 }
+
+.button-bubble {
+  z-index: 99;
+  position: fixed;
+  right: -0.5rem;
+  top: -0.5rem;
+}
```

Don´t you think the cropped button looks a bit weird ? What about something like this?

```suggestion
}.button-bubble {
  z-index: 99;
  position: fixed;
  right: -0.2rem;
  top: -0.2rem;
}

.button-bubble > .button-chip {
  border-radius: 0.5rem;
}
```
```

---

take a look at the latest proposition, please